src/arrayMethodJoin.js

```diff
@@ -4,8 +4,27 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
```

Comment:
```suggestion
    let joinedString = '';
```
str - погана назва для змінної